lib/plugins/filter/after_post_render/external_link.js

```diff
@@ -16,16 +16,12 @@ function externalLinkFilter(data) {
   if (config.external_link === false || config.external_link.enable === false
     || config.external_link.field !== 'post') return;
 
-  const exclude = Array.isArray(config.external_link.exclude) ? config.external_link.exclude
```

Comment:
https://github.com/hexojs/hexo-util/blob/c876271c05f509dc3d32aba9ec377db32e597997/lib/is_external_link.js#L28-L29

`isExternalLink()` has built-in cast array support.